sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Azure.Messaging.EventHubs.Processor.Samples.csproj

```diff
@@ -18,4 +18,8 @@
   <ItemGroup>
     <ProjectReference Include="..\src\Azure.Messaging.EventHubs.Processor.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Targets" PrivateAssets="all" />
```

Comment:
What is this needed for?

Reply:
I was following instructions [here](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605#example-2) to try and fix NuGet Error NU1605 that was occurring in Azure.Messaging.EventHubs.Processor.Samples

Reply:
I don't think this is the right solution. Can you show the exact error you were hitting?